### PR TITLE
Save users' map preferences

### PR DESCRIPTION
**Pull request by southbears**
Dework task details: https://app.dework.xyz/bounties?taskId=80f45755-5af8-4892-b840-6aaf71e5f05e

### CODE REVIEW

File: components/Map.tsx [hunk 1]
```diff
@@ -1,29 +1,29 @@
 import * as React from "react";
-import { useState, useEffect, useRef, useCallback } from "react";
```
Comment (southbears) [inline components/Map.tsx:2]: VSCode linting did this automatically. Let me know if you'd prefer me to revert this.

**Comment by gravenp** [comment]
@southbears can you reopen this PR against the latest version of the `develop` branch?

**Comment by codynhat** [comment]
@southbears Let's try to clean up the style differences. I added a prettier check to the CI. Can you run `yarn prettier` and push again?

**Comment by southbears** [comment]
> @southbears Let's try to clean up the style differences. I added a prettier check to the CI. Can you run `yarn prettier` and push again?

Done!

File: components/Map.tsx [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+import { useCallback, useEffect, useRef, useState } from "react";
 // eslint-disable-next-line import/named
+import { gql, useQuery } from "@apollo/client";
+import Col from "react-bootstrap/Col";
 import ReactMapGL, { MapEvent, NavigationControl } from "react-map-gl";
```
Comment (codynhat) [inline components/Map.tsx:6]: The linter is failing here now. I believe the comment above `// eslint-disable-next-line import/named` needs to be directly above this import.

**Review by gravenp — COMMENTED** [review]
Looks good on the user testing side. Thanks, @southbears!

@codynhat you can approve & merge if you agree.

**Pull request merged**